tools/worker/swift_runner.cc

```diff
@@ -475,55 +506,57 @@ void SwiftRunner::ProcessDiagnostics(std::string stderr_output,
   // semicolon (for wrapped diagnostics). Nothing guarantees this for the
   // wrapped case -- it is just observed convention -- but it is sufficient
   // while the compiler doesn't give us a more proper way to detect these.
-  std::regex const diagnostic_name_pattern{"\\[([_A-Za-z][_A-Za-z0-9]*)\\](;|$)"};
+  std::regex const diagnostic_name_pattern{"\\[([_A-Za-z][_A-Za-z0-9]*)\\]"};
 
   std::istringstream error_stream(stderr_output);
-  std::string line;    
+  std::string line;
   while (getline(error_stream, line, '\n')) {
-      std::unique_ptr<std::string> modified_line;
+    std::unique_ptr<std::string> modified_line;
```

Comment:
Would it be possible to avoid unrelated format/indent changes? May complicate rebasing upstream in the future

Reply:
👌